extensions/fairifier-rdf-extension/dtls/fairifier/DetectFileFormatCommand.java

```diff
@@ -134,21 +141,25 @@ public void doPost(HttpServletRequest req, HttpServletResponse res)
                 format = parsers[i].getRDFFormat().getDefaultMIMEType();
                 break;
             } catch (Exception e) {
-                System.out.println(
-                        parsers[i].getRDFFormat().getDefaultMIMEType() + " : " + e.toString());
                 continue;
             }
         }
-        try {
-            res.setCharacterEncoding("UTF-8");
-            res.setHeader("Content-Type", "application/json");
-            JSONWriter writer = new JSONWriter(res.getWriter());
-            writer.object();
-            writer.key("RDFFormat");
-            writer.value(format);
-            writer.endObject();
-        } catch (Exception e) {
-            respondException(res, e);
+        return format;
+    }
+
+    private String getFormatFromUrl(RDFParser[] parsers, String url, InputStream[] in)
+            throws IOException, MalformedURLException {
+        StringBuffer response = new StringBuffer();
+        URL obj = new URL(url);
+
+        for (RDFParser parser : parsers) {
+            HttpURLConnection con = (HttpURLConnection) obj.openConnection();
+            con.addRequestProperty("Accept", parser.getRDFFormat().getDefaultMIMEType());
```

Comment:
`"Accept"` can be a constant, if you have something like guava or spring accessible then use their `HttpHeaders` objects.